.github/workflows/main.yml

```diff
@@ -77,7 +78,7 @@ jobs:
         meson compile -C build/ --verbose
         meson install -C build/
 
-    - name: Test (Taisei)
+    - name: (Taisei)
```

Comment:
That's a weird name for a step. Overzealous substitution? Maybe call it "run tests" or something.